src/core/config/OperationConfig.js

```diff
@@ -3312,6 +3313,15 @@ const OperationConfig = {
             },
         ]
     },
+    "Extract EXIF": {
+        description: [
+        ].join("\n"),
```

Comment:
Add something useful here explaining what EXIF data is and which file formats might contain it. Remember that the description is checked when people use the search utility, so I usually try to include words here that people might search for when trying to find this operation.